swan-lake/integration-tutorials/service-orchestration.md

```diff
@@ -108,90 +108,89 @@ Follow the instructions given in this section to develop the service.
     $ bal new service-orchestration
     ```
 
```

Comment:
Shall we update L15 to say

"call the backend services to make the appointment"

instead of 

"retrieve details from the backend services"